.golangci.yml

```diff
@@ -27,6 +27,7 @@ linters:
   - whitespace
   disable:
   - scopelint
+  - testifylint # To many false positives https://github.com/Antonboom/testifylint/issues/67
```

Comment:
```suggestion
  - testifylint # Too many false positives https://github.com/Antonboom/testifylint/issues/67
```

Reply:
For posterity, can we enumerate some examples of the false positives in this comment for future code readers?

Reply:
I've updated the linked issue with the false positive examples - I'd prefer people refer to that